clang/test/Format/main-include-char-bracket-group.cpp

```diff
@@ -0,0 +1,27 @@
+// Test the combination of regrouped include directives, via regexes and
```

Comment:
Our tests reside in `clang/unittests/Format/SortIncludesTest.cpp`.

Reply:
Ok I will move them there, thanks :) Should I remove these lit tests or are they complementary?

Reply:
I've never run anything but the unit tests. I'd remove these, but I have no strong opinion on that matter.

Reply:
> Ok I will move them there, thanks :) Should I remove these lit tests or are they complementary?

Please remove them from the lit tests.

Reply:
Done.